src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/DependencyTelemetryConverter.cs

```diff
@@ -34,12 +34,13 @@ protected override DependencyTelemetry CreateTelemetryEntry(LogEvent logEvent, I
             string resultCode = logEntry.Properties.GetAsRawString(nameof(DependencyLogEntry.ResultCode));
             bool outcome = logEntry.Properties.GetAsBool(nameof(DependencyLogEntry.IsSuccessful));
             IDictionary<string, string> context = logEntry.Properties.GetAsDictionary(nameof(DependencyLogEntry.Context));
-            
+
+            string dependencyId = logEntry.Properties.GetAsRawString(ContextProperties.DependencyTracking.DependencyId);
             string operationId = logEvent.Properties.GetAsRawString(ContextProperties.Correlation.OperationId);
-            
+
             var dependencyTelemetry = new DependencyTelemetry(dependencyType, target, dependencyName, data, startTime, duration, resultCode, success: outcome)
             {
-                Id = operationId
+                Id = dependencyId ?? operationId
```

Comment:
Not sure if this is still correct, we shouldn't use the operation ID anymore, right?

Reply:
Yes, let's remove this, as the `OperationId` is already assigned in the operation context information of the telemetry.